### DISCUSSION TITLE
WriteStatus function

### DISCUSSION THREAD

**Discussion by sailingwithsandeep** [uNetworking/uWebSockets.js#1033] [General]
Writes the HTTP status message such as "200 OK".
     * This has to be called first in any response, otherwise it will be called automatically with "200 OK".

May i know why this was necessary?
What was your thought process behind it?
Sorry for being curious here.

**Comment by uNetworkingAB** [comment]
Because HTTP responses are written unbuffered and must be written in order

**Reply by sailingwithsandeep** [reply]
Gotcha
